book/types_of_data.md

```diff
@@ -155,7 +155,19 @@ true
 
 ## Ranges
 
-A range is a way of expressing a sequence of values from start to finish. They take the form \<start\>..\<end\>. For example, the range `1..3` means the numbers 1, 2, and 3.
+A range is a way of expressing a sequence of integer or float values from start to finish. They take the form \<start\>..\<end\>. For example, the range `1..3` means the numbers 1, 2, and 3.
+
+::: tip
+
+You can also easily create lists of characters with a form similar to ranges with the command [`seq char`](/commands/docs/seq_char.html) as well as with dates using the [`seq date`](/commands/docs/seq_date.html) command.
```

Comment:
maybe mention `seq char` too?

Reply:
I think it does. Or did you mean it should provide an example or explain them or something?

Reply:
lol, i have no idea what i meant. sorry for the noise.